### PR TITLE
This is the changes...

### PR DESCRIPTION
**Pull request by sandip-dak**

### CODE REVIEW

**Review by vikram-manohar — APPROVED** [review]

File: Sandip1/Controllers/StudentController.cs
Comment (sandip-dak) [inline Sandip1/Controllers/StudentController.cs:1]: done

File: Sandip1/Controllers/TeacherController.cs
Comment (sandip-dak) [inline Sandip1/Controllers/TeacherController.cs:1]: done...

**Review by sandip-dak — COMMENTED** [review]
Sandip.Roy@shell.com, done

**Comment by sandip-dak** [comment]
done

**Pull request closed without merging**